From 4c3b432383af6a328eb4c0e77b4148004c98a09e Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 2 Apr 2012 19:04:56 +0200
Subject: [PATCH 117/125] journal: decrease default mmap window size to allow
 a bigger number of journals to be traversed in
 parallel

Conflicts:

	TODO
---
 src/journal/journal-file.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/journal/journal-file.c b/src/journal/journal-file.c
index 5d540a7..e863a27 100644
--- a/src/journal/journal-file.c
+++ b/src/journal/journal-file.c
@@ -35,7 +35,7 @@
 #define DEFAULT_DATA_HASH_TABLE_SIZE (2047ULL*16ULL)
 #define DEFAULT_FIELD_HASH_TABLE_SIZE (2047ULL*16ULL)
 
-#define DEFAULT_WINDOW_SIZE (128ULL*1024ULL*1024ULL)
+#define DEFAULT_WINDOW_SIZE (8ULL*1024ULL*1024ULL)
 
 #define COMPRESSION_SIZE_THRESHOLD (512ULL)
 
-- 
1.7.9.3

